src/lib/components/Links.svelte

```diff
@@ -25,14 +27,29 @@
   // export const isFavorite = (url: string) => {
   //   return false;
   // };
+  const scrollLeft = () => {
+    const ul = document.querySelector("#linkList-ul");
+    ul?.scrollBy(-30, 0);
+  };
+  const scrollRight = () => {
+    const ul = document.querySelector("#linkList-ul");
+    ul?.scrollBy(30, 0);
+  };
 </script>
 
 <section class="links--container">
   <h2 class="links--title">
     <SvgIcon src={launch} />Vos sites publics
   </h2>
   <h3>Connexion vers les services publics</h3>
-  <ul class="links--list">
+  <button class="scroll--button" on:click={scrollLeft}>
+    <SvgIcon src={arrow_back} />
+  </button>
+
+  <button class="scroll--button" on:click={scrollRight}>
+    <SvgIcon src={arrow_forward} />
+  </button>
+  <ul class="links--list" id="linkList-ul">
```

Comment:
La classe ne suffisait pas pour le sélectionner ?

Reply:
Je préfère un id parce qu'on est sûr  qu'il y en a qu'un dans la page alors qu'avec une classe ça peut prendre un autre élément (c'est le cas si on ne change pas le nom de la classe de l'<ul />)

Reply:
L'id ne garantit pas qu'il soit unique. C'est une règle a respecter mais elle n'est pas garantie.

De la même manière la classe est unique si tu décides qu'elle le soit.

Les id servent à autre chose qu'à être des sélecteurs CSS. Aucun framework css-in-js ne les utilisent pour de bonnes raisons.

Une classe CSS est censée être réutilisable, composite et mixable. L'utilisation des id va à l'encontre de ça. De plus les sélecteurs via I'd sont prioritaires sur les autres ils empêchent (ou du moins rendent plus difficile) donc les compositions

https://github.com/CSSLint/csslint/wiki/Disallow-IDs-in-selectors